### DISCUSSION TITLE
Polish translation issue

### DISCUSSION THREAD

**Discussion by Skulmotyw** [diasurgical/DevilutionX#3968] [Q&A]
Hi i dont know if this is correct place to ask this question. I cant run diablo in polish. I downloaded game from gog downloaded devilutionX and polish  translation mpq however when I run it on PC it opens hellfire in English without possiibilty to switch language. Is there same setting this im omitting? Thanks for support. Regards

**Comment by qndel** [comment]
you have to set language to polish in %appdata%/diasurgical/diablo.ini
`[Language]
Code=pl`
